.renovaterc.json

```diff
@@ -13,6 +13,14 @@
       "versioning": "loose"
     }
   ],
+  "customDatasources": {
+    "VUIIS-dax": {
+      "defaultRegistryUrlTemplate": "https://api.github.com/repos/VUIIS/dax/contents/misc/xnat-plugins",
+      "transformTemplates": [
+        "{\"releases\": $.{\"version\": $join([\"v\", $match(name, /(\\d+\\.)?(\\d+\\.)?(\\*|\\d+)/ ).match ])}}"
```

Comment:
This transforms the HTTP response into the format required by renovate. I tried it on https://try.jsonata.org/ using the output from `curl https://api.github.com/repos/VUIIS/dax/contents/misc/xnat-plugins` and the query:
```jsonata
{"releases": $.{"version": $join(["v", $match(name, /(\d+\.)?(\d+\.)?(\*|\d+)/ ).match ])}}
```